python_modules/dagster/dagster/_core/execution/asset_backfill.py

```diff
@@ -1355,40 +1357,57 @@ def can_run_with_parent(
             and partition_mapping.end_offset == 0
         )
     )
-    return (
-        parent_node.backfill_policy == candidate_node.backfill_policy
-        and parent_node.priority_repository_handle is candidate_node.priority_repository_handle
-        and parent_node.partitions_def == candidate_node.partitions_def
-        and (
-            parent.partition_key in asset_partitions_to_request_map[parent.asset_key]
-            or parent in candidates_unit
+    if parent_node.backfill_policy != candidate_node.backfill_policy:
```

Comment:
I looked through the conversion of condition a bunch of times to make sure i got everything right, but more eyes to make sure i didn't flip anything is appreciated!